### ISSUE TITLE
Value return null

### ISSUE THREAD

**Issue by joumduk** [flutter-package/images_picker#14]
When i try to pick type all.

```
List<Media> res = await ImagesPicker.pick(
      count: 5,
      pickType: PickType.all,
      quality: 0.8,
      maxSize: 500,
    );
```

when used from IOS, 'res' is return null

**Comment by chavesgu** [comment]
Sorry, it's my code wrong.I will solve it. But I will deploy this problem with null-safety on new flutter version

**Comment by chavesgu** [comment]
It has been fixed in `1.0.0`
